Userland/Services/WindowServer/Button.h

```diff
@@ -52,6 +52,8 @@ class Button : public Weakable<Button> {
     void on_mouse_event(const MouseEvent&);
 
     Function<void(Button&)> on_click;
+    Function<void(Button&)> on_right_click;
+    Function<void(Button&)> on_aux_click;
```

Comment:
Let's just call this `on_middle_click` as that names is used everywhere else.

Reply:
Done.